notebook/app.py

```diff
@@ -116,6 +116,16 @@ def get_page_config(self):
         )
         return page_config
 
+    def server_extension_is_enabled(self, extension):
```

Comment:
Has to be broken into a separate function for get_page_config to be below Ruffs cyclomatic complexity limits